01 - Community/People/universvm.md

```diff
@@ -0,0 +1,63 @@
+---
+aliases:
+- Leonardo Castorina
+tags:
+- 
+publish: true
+---
+
+# Leonardo Castorina
+
+- Publish sites: <https://medium.com/@universvm/> ^website
+- Twitter: [Twitter](https://twitter.com/leocastorina/) ^twitter-->
```

Comment:
When previewed in Obsidian, this line looks odd.

![image](https://user-images.githubusercontent.com/4840096/201439439-c939ccc7-300e-4db4-920a-cbf73cf1041a.png)


Please remove the `-->` split comment at the end of the line. Thank you.